apps/herbatika/src/styles/tokens/_herbatika-spacing.css

```
@@ -56,6 +56,11 @@
   --spacing-product-detail-page: var(--spacing-page-default);
   --spacing-product-detail-page-lg: var(--spacing-page);
   --spacing-product-detail-page-gap: var(--spacing-page-gap);
+  --spacing-product-detail-information-scroll-offset: calc(
+    var(--height-header-mobile) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)